bigtable/admin.go

```diff
@@ -960,13 +1038,27 @@ type ClusterConfig struct {
 	InstanceID, ClusterID, Zone string
 	NumNodes                    int32
 	StorageType                 StorageType
+
+	// KMSKeyName describes the Cloud KMS encryption key that will be used to
+	// protect the destination Bigtable cluster. The requirements for this key
+	// are:
+	//  1) The Cloud Bigtable service account associated with the project that
+	//  contains this cluster must be granted the
+	//  `cloudkms.cryptoKeyEncrypterDecrypter` role on the CMEK key.
```

Comment:
It seems strange to have full docs only for this field? Also, "CMEK key" seems redundant 😄